CAP/gap/CategoryMorphisms.gd

```diff
@@ -170,6 +170,10 @@ DeclareProperty( "IsIdempotent",
 #! @Description
 #! The arguments are an object $a$ in a category $C$ and an integer $n$.
 #! The output is a random morphism $\alpha: a \rightarrow b$ for some object $b$ in $C$.
+#! If $C$ is equipped with the methods <C>RandomObjectByInteger</C> and <C>RandomMorphismWithFixedSourceAndRangeByInteger</C>,
```

Comment:
```suggestion
#! If $C$ is equipped with the methods <C>RandomObjectByInteger</C> and <C>RandomMorphismWithFixedSourceAndRangeByInteger</C>
#! and $C$ is an Ab-category,
```
Similarly for the other two derivations from RandomMorphismWithFixedSourceAndRangeByInteger.

Reply:
And of course also for the three derivations from RandomMorphismWithFixedSourceAndRangeByList.